mqtt/build/linux/test.sh

```diff
@@ -63,8 +63,14 @@ process_args()
 
 process_args "$@"
 
+# Get cargo2junit to report test results to Azure Pipelines
+$CARGO install cargo2junit
```

Comment:
lets move cargo2junit out of the test.sh script? Or at least cover this with command-line argument. For instance, someone may use this script to run all tests before he pushes the code, but he doesn't want prepare any reports locally